CMakeLists.txt

```diff
@@ -404,3 +405,19 @@ install (
 # configure the packaging stuff
 #
 include (CPackLists.txt)
+
+#
+# Generate the documentation using doxygen
+#
+if (BUILD_DOCUMENTATION)
+  message (STATUS "Generating documentation ...")
+  set (CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR})
+  set (DOXYFILE_SOURCE_DIR "${CMAKE_SOURCE_DIR}/cvmfs")
+  set (DOXYFILE_OUTPUT_DIR "${CMAKE_BINARY_DIR}")
+  set (DOXYFILE_LATEX "OFF")
+  execute_process (
+    COMMAND sh -c "cp ${CMAKE_SOURCE_DIR}/doc/Doxyfile.in ${CMAKE_SOURCE_DIR}/build"
+    COMMAND sh -c "/bin/sed -i 's/@VERSION@/${CernVM-FS_VERSION_STRING}/g' ${CMAKE_SOURCE_DIR}/build/Doxyfile.in"
```

Comment:
It's more secure to use `/bin/sed -i -e 's/@VERSION@/${CernVM-FS_VERSION_STRING}/g' ${CMAKE_SOURCE_DIR}/build/Doxyfile.in"